components/tooltip/tooltip.js

```diff
@@ -296,6 +304,11 @@ class Tooltip extends RtlMixin(LitElement) {
 		this._dismissibleId = null;
 	}
 
+	firstUpdated(changedProperties) {
+		super.firstUpdated(changedProperties);
+		this.setAttribute('role', 'tooltip');
```

Comment:
Wondering if we should NOT apply the role if `aria-describedby` isn't getting set? [The spec](https://www.w3.org/WAI/PF/aria/roles#tooltip) seems to imply that it's a requirement, but not sure if it matters.

Reply:
That sounds reasonable to me, reading into other sources it sounds like all `role="tooltip"` does is expand support for `aria-describedby` with some screen readers so its useless if `aria-describedby` isn't present.